### PR TITLE
feat(http-client-patch): add http client patch method

### PR DESCRIPTION
**Pull request by zverbeta**

### CODE REVIEW

**Review by grankovsky — DISMISSED** [review]

File: packages/components/entity-collection/i-entity-collection-pageable.ts [hunk 1]
```diff
@@ -43,6 +43,9 @@ export interface IEntityCollectionPageable<T extends IId = unknown, Extra extend
     /** PUT request. Use to replace or delete entity data. */
     update(update: Partial<T>, params?: QueryParams, headers?: HttpHeaders): Observable<HttpResponse<T>>;
 
+    /** PATCH request. Use to replace or delete entity data. */
```
Comment (Kaskyi) [inline packages/components/entity-collection/i-entity-collection-pageable.ts:46]: Why do we need a new method if the `update` method do the same thing ?

Comment (Bozonych) [reply packages/components/entity-collection/i-entity-collection-pageable.ts:46]: When we want update one field of entity:
`update` -  we must send all entity
`path` - we can send only updated field

**Review by Bozonych — DISMISSED** [review]

File: packages/components/entity-collection/i-entity-collection-pageable.ts (hunk 1 above)
Comment (Kaskyi) [reply packages/components/entity-collection/i-entity-collection-pageable.ts:46]: There are 2 points:
- tsdoc comment are same for update and patch.
- update and patch argument is Partial<T>.
In http there put and patch. In collection I suggest to change to `update(patch)` and `replace(put)`

**Review by sergeysenja1992 — DISMISSED** [review]

File: packages/components/entity-collection/i-entity-collection-pageable.ts (hunk 1 above)
Comment (sergeysenja1992) [reply packages/components/entity-collection/i-entity-collection-pageable.ts:46]: PUT and PATCH is different http method, PATCH very usefull. We plan to support it on entity BE too.

**Review by Sumragen — DISMISSED** [review]

**Pull request closed without merging**